examples/01_encodings.py

```diff
@@ -0,0 +1,195 @@
+
+feature_names = tv.get_feature_names_out()
+
+feature_names[4:82]
+
+###############################################################################
+#     - The |GapEncoder| for high cardinality string columns, 'employee_position_title'
+#       and 'division'. The |GapEncoder| is a powerful encoder that can handle dirty
+#       categorical columns.
+
+feature_names[82:]
+
+###############################################################################
+#     - The |DatetimeEncoder| to the 'date_first_hired' column. The |DatetimeEncoder|
+#       can encode datetime columns for machine learning.
+
+feature_names[0:4]
+
+###############################################################################
+# As we can see, it gave us interpretable column names.
+#
+# In total, we have a reasonable number of encoded columns:
+
+len(feature_names)
+
+###############################################################################
+# Let's look at the cross-validated R2 score of our model:
+
+from sklearn.model_selection import cross_val_score
+import numpy as np
+
+scores = cross_val_score(pipeline, X, y)
+print(f"R2 score:  mean: {np.mean(scores):.3f}; std: {np.std(scores):.3f}\n")
+
+###############################################################################
+# The simple pipeline applied on this complex dataset gave us very good results.
+
+###############################################################################
+# Feature importances in the statistical model
+# --------------------------------------------
+#
+# In this section, after training a regressor, we will plot the feature importances.
+#
+# .. topic:: Note:
+#
+#   To minimize computation time, we use the feature importances computed by the
+#   |RandomForestRegressor|, but you should prefer |permutation importances|
+#   instead (which are less subject to biases).
+#
+# First, let's train another scikit-learn regressor, the |RandomForestRegressor|:
+
+from sklearn.ensemble import RandomForestRegressor
+
+regressor = RandomForestRegressor()
+
+pipeline = make_pipeline(TableVectorizer(), regressor)
+pipeline.fit(X, y)
+
+###############################################################################
+# We are retrieving the feature importances:
+
+importances = regressor.feature_importances_
+std = np.std([tree.feature_importances_ for tree in regressor.estimators_], axis=0)
+indices = np.argsort(importances)
+# Sort from least to most valuable
+indices = list(reversed(indices))
+
+###############################################################################
+# And plotting the results:
+
+import matplotlib.pyplot as plt
+
+plt.figure(figsize=(12, 9))
+plt.title("Feature importances")
+n = 20
+n_indices = indices[:n]
+labels = np.array(feature_names)[n_indices]
+plt.barh(range(n), importances[n_indices], color="b", yerr=std[n_indices])
+plt.yticks(range(n), labels, size=15)
+plt.tight_layout(pad=1)
+plt.show()
```

Comment:
Nitpick, improves readability
```suggestion
top_indices = indices[:20]
labels = np.array(feature_names)[top_indices]

plt.figure(figsize=(12, 9))
plt.barh(
    y=labels,
    x=avg_importances[top_indices],
    yerr=std_importances[top_indices],
    color="b",
)
plt.yticks(fontsize=15)
plt.title("Feature importances")
plt.tight_layout(pad=1)
plt.show()
```